ob1k-cache/src/main/java/com/outbrain/ob1k/cache/memcache/folsom/MemcachedClient.java

```diff
@@ -160,6 +174,7 @@ private boolean isOK(final MemcacheStatus memcacheStatus) {
             consumer.consume(Try.fromError(e));
           } catch (final ExecutionException e) {
             final Throwable error = e.getCause() != null ? e.getCause() : e;
+            log.error("Failed to run operation for " + cacheName + ", error: " + error.getMessage(), error);
```

Comment:
Removed the error message.
Note that there's no overload of log.error that supports placeholders and logging the exception.